rbd.c

```diff
@@ -1159,9 +1159,11 @@ static int tcmu_rbd_open(struct tcmu_device *dev, bool reopen)
 #ifdef LIBRADOS_SUPPORTS_GETADDRS
 	/* Get current entry address for the image */
 	ret = rados_getaddrs(state->cluster, &state->addrs);
-	tcmu_dev_info(dev, "address: {%s}\n", state->addrs);
-	if (ret < 0)
-		return ret;
+	if (ret < 0) {
+		tcmu_dev_err(dev, "failed getting the address\n");
+		goto stop_image;
+	}
+	tcmu_dev_info(dev, "opened, the address: {%s}\n", state->addrs);
```

Comment:
Nit: keep the old string as the image is opened unconditionally whereas this message is printed only if `rados_getaddrs()` is supported (nautilus and later).
```suggestion
	tcmu_dev_info(dev, "address: {%s}\n", state->addrs);
```

Reply:
Will fix it.